src/test/scala/net/ruippeixotog/scalascraper/Examples.scala

```diff
@@ -17,6 +18,14 @@ object ExampleMatchers {
   val errs = validatorsAt[Int]("error-matchers")
 }
 
+object ProxyApp extends App {
+  ProxyUtils.setProxy("localhost", 3128)
+  val browser = Browser()
+  val doc = browser.get("http://observador.pt")
+
+  println("=== OBSERVADOR HTTP & HTTPS PROXY ===")
```

Comment:
Can you complete this example with the contents of some other scraping example, e.g. `NewsApp`? That would ensure users not only that the proxy is properly configured, but that it is really returning information.